From ce1b855b5766770ba330792eab1d1bd4662c210a Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 24 Jul 2022 19:22:45 +0300
Subject: [PATCH] [SW-0] ppboot: print final message to split output

Print a final ppboot message before loading another image on the CPU, so
the output will be cleaner and more understandable.

Change-Id: I76dac1d3dd5799b9ed4b56974ad9641933909989
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/ppboot/ppboot.c b/app/ppboot/ppboot.c
index e454585..34cbf92 100644
--- a/app/ppboot/ppboot.c
+++ b/app/ppboot/ppboot.c
@@ -313,6 +313,7 @@
 
 	ret = ppboot_load_preboot_img(&loadaddr);
 	ppboot_check_stack_monitor();
+	preboot_puts("Ppboot resigned\n\r\n");
 	if (ret)
 		image_start(loadaddr);
 	hl_enter_recovery();
